engine/Context.php

```diff
@@ -79,6 +79,7 @@ public function request( string $type ): bool {
 	 */
 	public function is_amp(): bool {
 		return \function_exists( 'is_amp_endpoint' ) && \is_amp_endpoint();
+		//return \function_exists( 'is_amp_request' ) && \is_amp_request();
 	}
```

Comment:
can you remove this? since it only works in WordPress backend i would say it is completely useless